src/CourseAuthoringRoutes.jsx

```diff
@@ -99,10 +99,7 @@ const CourseAuthoringRoutes = ({ courseId }) => {
             )}
         </PageRoute>
         <PageRoute path={`${path}/settings/grading`}>
-          {process.env.ENABLE_NEW_GRADING_PAGE === 'true'
-            && (
-              <GradingSettings courseId={courseId} />
-            )}
+          <GradingSettings courseId={courseId} />
```

Comment:
do we need to remove `ENABLE_NEW_GRADING_PAGE` from env either had already been removed?

Reply:
Removed